### ISSUE TITLE
Forward DateString method calls DateTime object?

### ISSUE THREAD

**Issue by caendesilva** [hydephp/hyde#284]
Would allow for `$post->date->format('Y-m-d')` instead of `$post->date->dateTimeObject->format('Y-m-d')`

**Comment by caendesilva** [comment]
@Cyberking99 sounds like a great plan! I've assigned you to this task so you can get right on working on this whenever it suits you. Here are the [Contribution Guidelines](https://github.com/hydephp/hyde/blob/1.x/CONTRIBUTING.md) which contain important practical information on how to set up the [`hydephp/develop`](https://github.com/hydephp/develop) where you'll submit your pull request. 

When you're ready for your submission to be reviewed, please change your PR from "draft" to "ready for review" and request a review from me. Remember to add tests!

Let me know if you have any questions, and have an amazing time with HydePHP! 🥳

**Comment by Cyberking99** [comment]
Hey @caendesilva, a quick question
Should I also update the existing classes (`RssFeedGenerator` and `MarkdownPost`) that implement this method call `$post->date->dateTimeObject->format('Y-m-d')` to  `$post->date->format('Y-m-d')` after implementing the magic method?

**Comment by caendesilva** [comment]
@Cyberking99 yeah that looks great! Feel free to do it in atomic commits to make it easier to review 🙌

**Comment by caendesilva** [comment]
Thank you so much @Cyberking99! This feature will roll out in this weekend's HydePHP v2 Release Candidate - in the meantime, feel free to apply to any other of the [open Hackathon issues](https://github.com/hydephp/hyde/issues?q=is%3Aissue%20state%3Aopen%20label%3AODHack14) - or play around with the latest release candidate to see if you can find any other improvements we need to make. If so we can create issues for that as well so you get more entries on the hackathon 🔥
